src/options.ts

```diff
@@ -1,35 +1,33 @@
 /// <reference path='../node_modules/@types/chrome/index.d.ts'/>
-"../node_modules/@types/chrome/index.d.ts";
+'../node_modules/@types/chrome/index.d.ts';
 // Saves options to chrome.storage
-function save_options(): any {
-	var defaultTab: HTMLCollectionOf<HTMLOptionElement> = (<HTMLSelectElement> document.getElementById("default-tab")).selectedOptions;
-	console.log(defaultTab[0].value);
+function save_options(): void {
+   var defaultTab: HTMLCollectionOf<HTMLOptionElement> = (<HTMLSelectElement>document.getElementById('default-tab')).selectedOptions;
```

Comment:
I would like to decide on a style to enforce across the entire repo. For now pretty much anything goes. Do you have any experience with linters?

Reply:
Yes actually. I'll add one, push it to this PR and tell me what you think.

Reply:
Ok - could it be enforced as a code check on PRs?